plugins/de.cognicrypt.staticanalyzer/src/de/cognicrypt/staticanalyzer/results/ResultsCCUIListener.java

```diff
@@ -118,9 +126,16 @@ public void analyzeDependenciesOnly(final Boolean depOnly) {
 	@Override
 	public void reportError(final AbstractError error) {
 		IResource sourceFile = null;
+		IPreferenceStore prefStore = Activator.getDefault().getPreferenceStore();
 		if (this.depOnly) {
 			return;
 		}
+		if (error instanceof ConstraintError) {
+			ConstraintError conError = (ConstraintError) error;
+			if (prefStore.getBoolean(Constants.SUPPRESS_LEGACYCLIENT_ERRORS) && conError.getBrokenConstraint().toString().equals("VC:protocol - TLSv1.2,") && isLegacyClient()) {
```

Comment:
For extensibility reasons: please put the `"VC:protocol - TLSv1.2,"` string into some class that holds this as an constant.